ros_foxglove_msgs/ros1/CameraCalibration.msg

```diff
@@ -62,3 +62,6 @@ float64[9] R
 # 
 # This holds for both images of a stereo pair.
 float64[12] P
+
+# Frame of reference for the camera. The origin of the frame is the optical center of the camera. +x points to the right in the image, +y points down, and +z points into the plane of the image.
+string frame_id
```

Comment:
Can we make this field first in the message?

Reply:
We would need to add special behavior for protobuf to keep binary compatibility. On the other hand, maybe we don't need to keep binary compatibility since the definitions will be embedded anyway?

Reply:
I don't feel strongly about it being first but it does seem like one of those nice things. Like if I was writing the message by hand I would want that.

If we are providing protobuf examples it seems nice to downstream consumers to attempt to be backwards compatible - tho we undid that with FrameTransform. We should be clear _somewhere_ very obvious of our goals with protobuf compatibility. Might be worth saying these schemas are provided as examples and that you should be mindful of field ids when making changes?